File: KeywordSuggestionsApi/src/main/java/hr/algebra/keywordsuggestionsapi/dto/KeywordSuggestionsDto.java
package hr.algebra.keywordsuggestionsapi.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class KeywordSuggestionsDto {
    private Integer id;
    private List<IdeaDto> ideas;
    private List<QuestionDto> questions;
}

File: ClientApp/ClientApp/Dto/KeywordSuggestionsDto.cs
namespace ClientApp.Dto
{
    public class KeywordSuggestionsDto
    {
        public int? Id { get; set; }
        public List<IdeaDto> Ideas { get; set; } = new();
        public List<QuestionDto> Questions { get; set; } = new();
    }
}
